Contrib/AffiliationServer/revenue.fsx

```diff
@@ -0,0 +1,118 @@
+#r "nuget:fstoolkit.errorhandling"
+#r "nuget:nbitcoin"
+#r "../../WalletWasabi/bin/Debug/net7.0/WalletWasabi.dll"
+#r "System.Security.Cryptography.dll"
+
+open System.Security.Cryptography
+
+open System.Text
+open NBitcoin
+open NBitcoin.RPC
+open System
+open System.IO
```

Comment:
Is it okay to keep `using`/`open` unordered? I noticed that in some files, but in most of them namespaces are ordered.

Reply:
It is okay in this case.